docs/config-file/v2.rst

```diff
@@ -88,6 +88,10 @@ Example:
       # Build all formats
       formats: all
 
+.. note::
+
+   PDF/epub output is not supported when using MkDocs  
```

Comment:
Actually `htmlzip` is not available either, we should put something like download are not currently supported for mkdocs or something like that

Reply:
got it

Reply:
I've currently added htmlzip to the note